src/components/Filters/NamespaceFilter.ts

```diff
@@ -3,7 +3,7 @@ import { authentication } from '../../utils/Authentication';
 import { FILTER_ACTION_APPEND, FilterType } from '../../types/Filters';
 
 export class NamespaceFilter {
-  static id = 'namespace';
+  static id = 'namespaces';
```

Comment:
@aljesusg I think you can remove the whole class & file. It is legacy code now, as far as i can tell it's now only used in `ListPageLink` just to get the `id` and `category` constants, so it doesn't worth it keeping as separate thing.

Reply:
it is using in the istioconfig list breadcumb too
Let me check the code